manager/assets/modext/widgets/core/modx.tabs.js

```diff
@@ -21,6 +21,58 @@ MODx.Tabs = function(config) {
     });
     MODx.Tabs.superclass.constructor.call(this,config);
     this.config = config;
+    this.on({
+        afterrender: function(tabPanel) {
+            if (MODx.request && MODx.request.hasOwnProperty('tab')){
+                const tabId = parseInt(MODx.request.tab);
+                this.setActiveTab(tabId);
+            }
+            /* Placing listener here because we only want to listen after the initial panel has loaded */
+            tabPanel.on({
+                beforetabchange: function(tabPanel, newTab, currentTab) {
+                    /*
+                        Only proceed with the clearing process if the tab has changed.
+                        This is needed to prevent clearing when a URL has been typed in.
+
+                        NOTE: The currentTab is the previous one being navigated away from
+                    */
+                    if (newTab.id != currentTab.id) {
```

Comment:
Installing pdoTools

![image](https://user-images.githubusercontent.com/5341208/214669925-6b04294f-bcb5-4865-9df2-e43bfea015a6.png)

Reply:
That's already fixed in #16360.